app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -63,22 +62,19 @@ fun SettingsScreen(modifier: Modifier = Modifier, viewModel: SettingsViewModel =
     val networkType by viewModel.updaterNetworkType.collectAsState(NetworkType.UNMETERED.name)
 
     Column(
-        modifier
-            .padding(horizontal = 20.dp)
+        modifier = modifier
+            .padding(horizontal = 8.dp)
             .verticalScroll(rememberScrollState()),
+        verticalArrangement = Arrangement.spacedBy(12.dp),
     ) {
         SettingGroupLabel(stringResource(R.string.customization), Modifier.padding(top = 16.dp))
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
             Setting(
                 label = stringResource(R.string.dynamic_color),
                 description = stringResource(R.string.dynamic_color_desc),
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .toggleable(
```

Comment:
I think there may be a problem with consolidating `toggleable()` into `clickable()` because `toggleable()` has [semantics](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier#(androidx.compose.ui.Modifier).toggleable(kotlin.Boolean,kotlin.Boolean,androidx.compose.ui.semantics.Role,kotlin.Function1)) which make it visible as a toggle to accessibility services. Without further information I'm hesitant to merge this in case it causes a regression for users using accessibility services.

@PatrykMis I'm curious whether you have any thoughts on this.

Reply:
Afaik, the only part that dictates what gets delivered to accessibility services is the Role.

But you're right, I'll fix it so you could customize it for each setting.

Reply:
Just pushed some changes to the settings roles. A role is required for a more general `Setting` and is hardcoded for specialized components like `Role.Switch` to `SwitchSetting`